src/config/types.rs

```diff
@@ -22,6 +22,7 @@ arg_enum! {
         Stdout,
         Xml,
         Html,
+        Info,
```

Comment:
If lcov only has one report format can you rename this to `Lcov` otherwise change it to something like `LcovInfo` (I'd prefer `Lcov` if it's clear enough)